pkg/proto/nbcontract/v1/kubeletconfig.proto

```diff
@@ -10,13 +10,10 @@ message KubeletConfig {
   repeated Taint startup_taints = 4;
   KubeletDiskType kubelet_disk_type = 5;
 
-  // kubelet_config_file_status specifies if the kubelet_config_file_content should be applied. 
-  FeatureState kubelet_config_file_status = 6;
```

Comment:
context for why this can be removed? is it always true? always false?

Reply:
Though I removed it from the contract, it's computable in the cse.cmd.sh. Please see https://github.com/Devinwong/karpenter-provider-azure/blob/3b7f1b27926597d3d3a55a80a561ee118951c86d/pkg/providers/imagefamily/bootstrap/cse_cmd_nbcontract.sh.gtpl#L99